code/modules/hydroponics/trays/tray_reagents.dm

```diff
@@ -85,35 +85,87 @@
 // *************************************
 
 /obj/item/reagent_containers/glass/fertilizer
-	name = "fertilizer bottle"
-	desc = "A small glass bottle. Can hold up to 60 units."
+	name = "jug"
+	desc = "A decent sized plastic jug. Can hold up to 80 units."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle16"
-	flags = OPENCONTAINER
-	possible_transfer_amounts = null
-	w_class = ITEMSIZE_SMALL
+	icon_state = "plastic_jug"
+	item_state = "plastic_jug"
+	flags = 0
+	possible_transfer_amounts = list(5, 10, 20, 40, 80)
+	volume = 80
+	w_class = ITEMSIZE_NORMAL
+
+	hitsound = 'sound/weapons/jug_empty_impact.ogg'
+	drop_sound = 'sound/weapons/jug_empty_impact.ogg'
 
-	//Like a shot glass!
-	amount_per_transfer_from_this = 10
-	volume = 60
+	fragile = 0
+	shatter = FALSE
+	unacidable = FALSE
+
+	force = 1
+	throwforce = 1
 
 /obj/item/reagent_containers/glass/fertilizer/Initialize()
 	. = ..()
 
 	src.pixel_x = rand(-5.0, 5)
 	src.pixel_y = rand(-5.0, 5)
 
+/obj/item/reagent_containers/glass/fertilizer/on_reagent_change()
+	. = ..()
+	update_icon()
+	if(reagents.total_volume)
+		var/fraction = reagents.total_volume / volume
+		force = max(10 * fraction, 1)
+		throwforce = max(10 * fraction, 1)
```

Comment:
Not sure about the specific values but I like this kind of calculation. We should have this for all reagent containers!

Reply:
Agreed, maybe minimum force and maximum force, then have it scale with the fraction of reagents. Also vars for a filled hitsound/drop_sound, with the default as the empty? :thinking:

Reply:
(Though, god. This goes down the thorny path of needing densities for reagents, because 10u of water and 10u of lead should probably have different forces.)

Reply:
Oh yeah. And I didn't want to scare Geeves in the other comment but the two variables should actually have different relation to the fullness (or I guess weight) of the container: things _do_ make more damage on impact the heavier they are, but they also get harder to throw.

Actually it's even more complicated than that since for every object there's a different point where it's easiest to throw but heavy enough to do damage; in the end this is what our force and throwforce vars do and why they are set and "balanced" manually.

For the reagents container a (fairly) simple but still kinda realistic solution could be to define a different "middle point" for every object size; tiny objects would be easiest to throw when completely full, small objects when maybe 3/4 full, large objects only when half full, etc. We could even have this with some tweak variable to facilitate for different object shapes/grips/etc., though that's probably not necessary.

The force could remain with an equation like the above, as that's more or less linear relationship (I think?).

Reply:
im just gonna lower the max throwforce to 8 then